modules/y_lb_main_menu_cta_block/y_lb_main_menu_cta_block.install

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @file
+ * Install, update and uninstall functions for the Website Services Main Menu CTA block module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function y_lb_main_menu_cta_block_install(): void {
+  \Drupal::messenger()->addStatus(t('Module Website Services Main Menu CTA block has been installed.'));
```

Comment:
Why do we need this?
Core is doing such messages
![image](https://github.com/user-attachments/assets/0e360a7d-1289-4ded-a74a-0913826c65ec)

Reply:
Those were from `drush generate install-file` and I'd just left them. Thanks for the feedback.